src/pages/DenyRoot.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <h2 v-if="isUnix">Cannot Run As Root</h2>
+    <h2 v-else>Cannot Run As Administrator</h2>
+    <p v-if="isUnix">
+      Rancher Desktop cannot be run with root privileges.
+      Please run again as a regular user.
+    </p>
+    <p v-else>
+      Rancher Desktop cannot be run as administrator.
+      Please run again as a regular user.
+    </p>
```

Comment:
I think we can simplify the template via computed props, for example:

```suggestion
    <h2>
      {{ message }}
    </h2>
    <p>
      {{ detail }}
      Please run again as a regular user.
    </p>
```

and then we could introduce the corresponding computed props to display the correct message:

```
    message(): string {
      return `Cannot Run as ${ this.isUnix ? 'Root' : 'Administrator' }`;
    },
    detail(): string {
      return `Rancher Desktop cannot be run ${ this.isUnix ? 'with root privileges' : 'as administrator' }.`;
    },
```